hoomd/md/force.py

```diff
@@ -376,6 +363,21 @@ def __init__(self, filter, rotation_diff=0.1):
 
         self._extend_typeparam([active_force, active_torque])
 
+    def _getattr_param(self, attr):
+        if self._attached:
+            if attr == "manifold_constraint":
+                return self._param_dict["manifold_constraint"]
+            parameter = getattr(self._cpp_obj, attr)
+            return parameter
+        else:
+            return self._param_dict[attr]
+
+    def _setattr_param(self, attr, value):
+        if attr == "manifold_constraint":
+            raise AttributeError(
+                "Cannot set manifold_constraint after construction.")
```

Comment:
I wonder if it would make sense/be plausible to remove and recreate the associated objects post construction?

Reply:
Usually it is not a good idea to switch constraints after initialisation as the forces become discontinuous. The only thing I can imagine, users might be interested in, is starting with a constraint and lifting it later. But this means detatching the restricted active force from the simulationa and attaching the unrestricted one afterwards anyway.